Translations/cs.lproj/InternetAccessPolicy.strings

```diff
@@ -13,5 +13,5 @@
 "ApplicationDescription" = "Keka je archivační aplikace pro macOS.";
 
 /* Software update connection: */
-"SoftwareUpdatePurpose" = "Keka vyhledává aktualizace připojením k u(pdate).keka.io.";
+"SoftwareUpdatePurpose" = "Keka vyhledává aktualizace pomocí připojení k serveru .keka.io.";
```

Comment:
The actual server is _u.keka.io_ or _update.keka.io_, that's why it must say _u(pdate).keka.io_

Reply:
OK, I'll fix it.

Reply:
Thanks again @ferben!